style.css

```diff
@@ -73,19 +88,260 @@ header {
   color: #344563;
 }
 
-.lets-connect {
+.social-list {
+  display: flex;
+  gap: 1.5rem;
+}
+
+.social-list li {
+  list-style: none;
+}
+
+.project1-h1 {
   font-family: "Poppins", sans-serif;
   font-style: normal;
+  font-weight: 700;
+  font-size: 3.2rem;
+  line-height: 4.4rem;
+  color: #172b4d;
+}
+
+.card-li {
+  display: flex;
+  list-style: none;
+  justify-content: space-between;
+  margin-right: 10.8rem;
+  gap: 0.8rem;
+}
+
+.card-li li {
+  padding: 0.4rem 1.2rem;
+  background: #ebebff;
+  border-radius: 0.8rem;
+  font-family: "Poppins", sans-serif;
+  font-size: 1.2rem;
   font-weight: 500;
-  font-size: 1.6rem;
-  color: #7f8cff;
+  line-height: 1.6rem;
+  letter-spacing: 0.03em;
+  color: #6070ff;
 }
 
-.social-list {
+.card {
   display: flex;
-  gap: 1.5rem;
+  flex-direction: column;
+  justify-content: space-between;
+  background-color: #fff;
+  padding: 1.6rem;
+  border: 1px solid #dfe1e6;
+  border-radius: 1.6rem;
+  gap: 2rem;
 }
 
-.social-list li {
+.card-div {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-right: 7.5rem;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 1.3rem;
+  line-height: 1.6rem;
+  color: #7a869a;
+}
+
+.dot {
+  border-radius: 100%;
+  width: 0.8px;
+  height: 0.8px;
+  background: #c1c7d0;
+}
+
+.canopy {
+  color: #344563;
+}
+
+.card-btn {
+  width: 12.4rem;
+  height: 4.8rem;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 1.7rem;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  border: 1px solid #6070ff;
+  border-radius: 0.8rem;
+  background: #fff;
+}
+
+.btn-getMyResume {
+  width: 16.5rem;
+  height: 4.8rem;
+  padding: 1.2rem;
+  border: 1px solid #6070ff;
+  border-radius: 0.8rem;
+  font-family: "Poppins", sans-serif;
+  font-weight: 500;
+  font-size: 1.7rem;
+  line-height: 2.4rem;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  background: #fff;
+}
+
+.about-mySelf {
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  padding: 0 2.4rem;
+  border-radius: top;
```

Comment:
- [ ] Great job with your styling 👏 . Please make sure you give the about section a `border-radius` so that it looks as close as possible to the Figma design

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/177837531-f9b7270f-412f-4dba-ad26-5a2e3d57d1da.png)  |  ![image](https://user-images.githubusercontent.com/74814780/177837555-9c9ddc09-08b3-4a99-a435-384ac88b99ed.png)

Reply:
OK I'll do that..thanks